### ISSUE TITLE
The app only works in split mode in android 14

### ISSUE THREAD

**Issue by Fiigfer** [Vita3K/Vita3K-Android#489]
I've tried a lot of different GPU drivers, but it only stays open in split screen, the instant I try to run it full screen it crashes.

**Comment by PangInFN** [comment]
It's the same problem. I noticed that even with split screen the app doesn't work in landscape orientation.

**Comment by NoMorePremium** [comment]
From Discord:

The devs don't own any A14 devices. 

Unfortunately there's no luck this time around. :(

**Comment by Pasis01** [comment]
Right, works only on split screen S23 Ultra Android 14.

**Comment by NoMorePremium** [comment]
Slight update:

Changing the driver to OpenGL will make it not crash, but when running games the screen will be black. Sound is present. 
I tried running the app through Winlator but unfortunately no games load. I was successful in installing the app though. 

I guess for A14 users, Vita3k is a no go for the present future. 

Finally: I've tried to use varioua Virtual Machines but again, no luck whatsoever.
